lib/shim/datastore-shim.js

```diff
@@ -608,59 +610,7 @@ function _getSpec({ spec, shim, fn, fnName, args }) {
  */
 function _recordQueryMetrics(parsed, segment, scope) {
```

Comment:
this function doesn't make sense. I would update where this is getting called to 

```js
segDesc.recorder = recordQueryMetrics.bind(parsed)
```

and then remove this function

Reply:
Got it